### DISCUSSION TITLE
[Suggestion] - A little edition of the mod!

### DISCUSSION THREAD

**Discussion by DarketzOficial** [Monika-After-Story/MonikaModDev#7816] [general - submod / external content]
_I just did a little editing on the mod Monika After Story!
I thought it would be better to post here to find out ... if you liked it!_
![image](https://user-images.githubusercontent.com/37710265/57665411-1d7faf80-75d2-11e9-8cc2-b9da8c21cf66.png)

(mod language in Portuguese!)
Made by Darketz.
https://www.reddit.com/user/Darketz

**Comment by Finikyu** [comment]
Language translation is neat, setting seems cool but the colours are way too high contrast. :S

Is the water animated or static?

**Comment by DarketzOficial** [comment]
> Language translation is neat, setting seems cool but the colours are way too high contrast. :S
> 
> Is the water animated or static?

Yes, the ocean is static! but I intend to cheer you up with the time I'm creating.

Thank you for liking the Portuguese language.

**Comment by multimokia** [comment]
Don't really recall when Monika became a redhead. But nice submod. Feel free to link the translation/the submod (if you intend to release it) to #3765

**Comment by AsuraShun** [comment]
One does not simply edit perfection
The background is neat but changing her hair color is a no

**Comment by DarketzOficial** [comment]
![monika_day_room](https://user-images.githubusercontent.com/37710265/57665777-a814de80-75d3-11e9-851f-3add67eef42d.png)
![monika_room](https://user-images.githubusercontent.com/37710265/57665760-9af7ef80-75d3-11e9-81a8-ccf48a9f6e0f.png)
![image](https://user-images.githubusercontent.com/37710265/57665823-d85c7d00-75d3-11e9-9182-d5ad769963c7.png)
![image](https://user-images.githubusercontent.com/37710265/57665806-c7137080-75d3-11e9-9508-8a28ca5ceadb.png)
Made by Darketz

I have here the art project I did in photoshop.
I assembled each part separately.
So I was putting the layers together little by little.
I used png images from the internet, like the clouds, the full moon and the wooden floor I made using textures.
And if it is for me to create it again separately, I will do it without problem!
![image](https://user-images.githubusercontent.com/37710265/57667522-37bd8b80-75da-11e9-8f02-d3f2dc3aeaf8.png)

**Comment by Abc2557** [comment]
About the hair: I don't see anything different... It looks the same color as the Monika I know and love. :|
